src/bindings/python/src/pyopenvino/graph/ops/parameter.cpp

```diff
@@ -55,6 +55,17 @@ void regclass_graph_op_Parameter(py::module m) {
 
     parameter.def_property("layout", &ov::op::v0::Parameter::get_layout, &ov::op::v0::Parameter::set_layout);
 
+    parameter.def(
+        "__eq__",
+        [](const ov::op::v0::Parameter& a, const ov::op::v0::Parameter& b) {
+            return a.get_instance_id() == b.get_instance_id();
+        },
+        py::is_operator());
```

Comment:
It changes behaviour of `==`, but `!=` is still overwritten by `ov.op.not_equal`, which can lead to inconsistency:
https://github.com/openvinotoolkit/openvino/blob/68f7d336e5a5ffaec88344b7f2eb36738b0306a4/src/bindings/python/src/openvino/runtime/__init__.py#L80-L81